checklists/lz_checklist.en.json

```diff
@@ -1432,6 +1432,10 @@
       "name": "Fulfilled",
       "description": "This check has been verified, and there are no further action items associated to it"
     },
+    {

```

Comment:
@fskelly  I believe this code block was accidentally pushed as it appears to be a duplicate of lines 1439-1441.